### ISSUE TITLE
Oath 2.0 is deprecated for new client ids

### ISSUE THREAD

**Issue by rweidman** [GoogleChromeLabs/web-vitals-report#20]
Can no longer use this for new projects, needs to be updated to use the newer identify auth Google is pushing.

**Comment by tunetheweb** [comment]
Can you explain a little more? I've not heard of this deprecation (though don't work in that area) and just created a new one fine, without any notice.

Also are you developing using this? As you should be able to access it from https://web-vitals-report.web.app/ without any OAUTH keys.

**Comment by rweidman** [comment]
Screenshot below of the warning on web-vitals-report. 

The error I get is:

`You have created a new client application that uses libraries for user authentication or authorization that will soon be deprecated. New clients must use the new libraries instead; existing clients must also migrate before these libraries are deprecated. See the [Migration Guide](https://developers.google.com/identity/gsi/web/guides/gis-migration) for more information.`

I misspoke when I said Oauth 2 was being deprecated.   What is being deprecated is the current sign in library, and that is being replaced by Google Identity Services.  New Client IDs do not work with the old library.

![Screen Shot 2022-08-30 at 4 29 05 PM](https://user-images.githubusercontent.com/36079725/187546842-5a8f49fb-3a27-4338-b0fa-3a7e68d8241c.png)

![Screen Shot 2022-08-30 at 4 34 13 PM](https://user-images.githubusercontent.com/36079725/187547860-9890a840-e022-4376-9612-c6e92f048e0a.png)

**Comment by tunetheweb** [comment]
I do not receive that error on either a local build nor on https://web-vitals-report.web.app/

I tried various versions of Chrome and also MS Edge.

Are you using any plugins like password managers?

**Comment by rweidman** [comment]
Strange, I get it in every browser I've tried.  I disabled all plugins on chrome and i still see it, as well as Firefox.  It's a warning, so it may not be showing up by default depending on your settings.

**Comment by tunetheweb** [comment]
Oooh you mean you see it in DevTools console? Not as an alert in the app?

I was confused with your "Can no longer use this" and screenshot. So it CAN still be used until that is removed in May 2023?

**Comment by rweidman** [comment]
Yes and yes.  DevTools console and there won't be an issue with Client Ids created in the past until March 2023.   I was just trying to implement a version of this locally, and it won't work with new Client Ids.  I have another account that is older that I might use for now.   Thanks!

**Comment by tunetheweb** [comment]
Yeah I can see new OAUTH tokens aren't accepted for the old API.

I've a branch with migration to GIS in #21 . Do you want to give it a go and see if that resolves it for you before we merge?
